tools/cleanup-build.py

```diff
@@ -63,12 +63,14 @@ def destroy(deployment_folder: str):
 
     if process.returncode == 0:
```

Comment:
I'd reverse the if/else and use `if process.returncode`.  You also don't actually need the else statement then, it just becomes the assumed default and you're just handling errors.

Reply:
Done.